Source/WebCore/platform/graphics/MediaPlayerPrivate.cpp

```diff
@@ -62,6 +62,14 @@ auto MediaPlayerPrivateInterface::asyncVideoPlaybackQualityMetrics() -> Ref<Vide
     return VideoPlaybackQualityMetricsPromise::createAndReject(PlatformMediaError::NotSupportedError);
 }
 
+MediaTime MediaPlayerPrivateInterface::apparentMediaTime() const
+{
+    auto aboutToSeekTime = this->aboutToSeekTime();
+    if (aboutToSeekTime.isValid())
+        return aboutToSeekTime;
+    return currentMediaTime();
+}
```

Comment:
It seems like this is just exposing an implementation detail. Why not have `MediaPlayerPrivateInterface:: currentMediaTime` do this automatically?

Reply:
I didn't want to introduce a regression, so this method is only called from the MediaSource, for the HTMLMediaElement it uses the same time type as before.

Reply:
I don't have any great suggestions, but the name is confusing

Reply:
It isn't great, but maybe `currentOrPendingSeekTime`?